React/tic-tac/src/app/components/Login/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment, Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import style from './style.scss';
+
+class Login extends Component {
+  state = {
+    loggedIn: false
+  };
+
+  loginHandle() {
+    this.setState(() => ({ loggedIn: true }));
+  }
+
+  fragmentRender() {
```

Comment:
You can move this logic to the render method using a ternary expression to know if login should or should not be redirected. 🙂